Adds new Advanced Behavioral Detections section (#4296)
elastic/security-docs: docs/advanced-entity-analytics/entity-risk-scoring.asciidoc

[[entity-risk-scoring]]
= Entity Risk Scoring

beta::[]

Entity Risk Scoring is an advanced {elastic-sec} analytics feature that helps security analysts detect changes in an entity's risk posture, hunt for new threats, and prioritize incident response.

> Entity Risk Scoring allows you to monitor the change in the risk posture of hosts and users from your environment. The risk scoring engine generates these advanced scoring analytics by factoring threats from its end-to-end XDR use cases, such as SIEM, cloud, and endpoint.

Reviewer: ```suggestion
Entity Risk Scoring allows you to monitor changes in the risk score of hosts and users in your environment. The risk scoring engine assesses risk based on data from various sources.
```
The sentence with "factoring" wasn't quite holding together. I'm not sure what all sources it uses so I just suggested various.

Author: How does this sound @benironside, @SourinPaul?  
```suggestion
Entity Risk Scoring allows you to monitor changes in the risk score of hosts and users in your environment. When generating advanced scoring analytics, the risk scoring engine takes into account threats from its end-to-end XDR use cases, such as SIEM, cloud, and endpoint.
```

Reviewer 2: @natasha-moore-elastic this is good, just one tweak. 
```suggestion
Entity Risk Scoring allows you to monitor risk score changes of hosts and users in your environment. When generating advanced scoring analytics, the risk scoring engine utilizes threats from its end-to-end XDR use cases, such as SIEM, cloud, and endpoint.
```

Reviewer 3: Reads great!

> The next-generation risk scoring engine provides greater scalability and performance. It leverages the Elastic SIEM detection engine to generate host and user risk scores from the last 30 days.

Reviewer: ```suggestion
It leverages the Elastic SIEM detection engine to generate host and user risk scores from the last 30 days.
```
Maybe I'm overzealous but the first sentence seemed pretty marketing to me. The remaining sentence could go at the beginning of the next paragraph.

> It also generates risk scores on a recurring interval, and allows for easy onboarding and management. The engine is built to factor in risks from all {elastic-sec} use cases, and allows you to customize and control how and when risk is calculated.

Reviewer: ```suggestion
It also generates risk scores on a recurring interval, and allows for easy onboarding and management. The engine is built to factor in risks from all {elastic-sec} use cases, and allows you to customize and control how and when risk is calculated.
```
"data from... use-cases" seems off to me. Maybe replace with "Data from various sources" or "Data from across SIEM"?

Author: I'm not sure which sentence this suggestion refers to?
